frontend/src/components/partners/partnersActivity.js

```diff
@@ -0,0 +1,130 @@
+import React, { useEffect, useState } from 'react';
+import ReactPlaceholder from 'react-placeholder';
+import PartnersProgresBar from './partnersProgresBar';
+import messages from './messages';
+import { OHSOME_STATS_BASE_URL } from '../../config';
+import { FormattedMessage } from 'react-intl';
```

Comment:
Also fix ordering of imports here and add a space between the imports and code

Reply:
@royallsilwallz code styling improvements like this one will be on our side.